prinz/src/main/scala/pl/touk/nussknacker/prinz/mlflow/repository/MLFBucketRepository.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.prinz.mlflow.repository
+
+import java.io.InputStream
+
+import pl.touk.nussknacker.prinz.mlflow.MlflowConstants.{S3_ACCESS_KEY, S3_SECRET_KEY, S3_URL}
+import pl.touk.nussknacker.prinz.util.amazon.S3Client
+
+class MLFBucketRepository(val bucketName: String) {
```

Comment:
I think we don't need `MLF` in names when classes come from `...mlflow.repository`.

Reply:
I think it is needed beacuse when you use these classes outside these packages naming may be confusing when you have many implementations of repositories